crd-generator/api/src/test/java/io/fabric8/crd/generator/v1/JsonSchemaTest.java

```diff
@@ -154,6 +154,13 @@ void shouldAugmentPropertiesSchemaFromAnnotations() throws JsonProcessingExcepti
     assertNull(defaultValue.getMaximum());
     assertNull(defaultValue.getPattern());
 
+    final JSONSchemaProps defaultValue2 = spec.get("defaultValue2");
+    assertEquals("my-value2", YAML_MAPPER.writeValueAsString(defaultValue2.getDefault()).trim());
+    assertNull(defaultValue.getNullable());
+    assertNull(defaultValue.getMinimum());
+    assertNull(defaultValue.getMaximum());
+    assertNull(defaultValue.getPattern());
```

Comment:
I might be wrong here but shouldn't you see asserting fields for `defaultValue2` here?

Reply:
good catch! fixed!